client/packages/coldchain/src/Equipment/DetailView/AppBarButtons.tsx

```diff
@@ -32,14 +32,13 @@ export const AppBarButtonsComponent = () => {
   };
 
   const printAssetLabel = () => {
+    const date = new Date().toLocaleString().split(', ')[0];
     setIsPrinting(true);
     fetch(Environment.PRINT_LABEL_QR, {
       method: 'POST',
       body: JSON.stringify({
         code: data?.id,
-        message: `${t('label.serial')}: ${data?.serialNumber ?? ''}\n${t(
-          'label.asset-number'
-        )}: ${data?.assetNumber ?? ''}`,
+        message: `${t('label.serial')}: ${data?.serialNumber ?? ''}\n${t('label.asset-number')}: ${data?.assetNumber ?? ''}\n${t('label.date-printed')}: ${date}`,
```

Comment:
```suggestion
        message: `${t('label.asset-number')}: ${data?.assetNumber ?? ''}\n${t('label.date-printed')}: ${date}`,
```
After discussion with @adamdewey we think it's best just to have the asset number as they're probably going to be set to the same thing, and serial number should already be on the device